### ISSUE TITLE
Using this with HMR

### ISSUE THREAD

**Issue by f0rr0** [dminkovsky/radium-loader#4]
I am trying to setup a dev environment with webpack. Here's the repo so far: https://github.com/sidjain26/react-es6-seed/tree/hmr Having issues in getting hmr to work. I am using radium-loader to load .css to inline styles. However, I get the error: radium-loader/index.js didn't return a function

**Comment by dminkovsky** [comment]
I don't know what you mean.

I:
- cloned your repo
- `npm install`
- `node_modules/.bin/webpack`.

It built `dist/bundle.js` which contains `app/components/RecentTracks/style.css` as a Radium object:

```
/* 211 */                                                                                                                                                                                                   
/***/ function(module, exports) {                                                                                                                                                                           

    module.exports = {"base":{"fontFamily":"monospace","backgroundColor":"#BADA55"}};                                                                                                                       

/***/ } 
```

So I am not seeing a problem.

---

As a side note, as I think you mentioned, instead of using `base` you are using `.base`:

```
<div style={styles['.base']}>
```

**Comment by dminkovsky** [comment]
Running `npm run` on the `hmr` branch produces no errors.

**Comment by f0rr0** [comment]
Try making changes to the style.css They don't reflect live in the browser 

<img width="810" alt="screen shot 2016-02-08 at 6 52 02 am" src="https://cloud.githubusercontent.com/assets/8574219/12888802/9af4bc8c-ce30-11e5-8132-ab7da5b8f50c.png">

**Comment by khadorkin** [comment]
Hi!
How is going with this? :bug:

**Comment by f0rr0** [comment]
Hi!

I experimented initially but I could not get it to work with some other
tools I was using. Inline styles with radium don't support
cq-prolyfill (container query). There is an implementation specifically for
React but I haven't tested it yet. Also, for small projects like the one I
was working on, inline styles did not give me any advantage over using the
traditional external css files. Also with css modules, I can keep my
css files local to the component I am working on. So, for now, I am still
using external modularized stylesheets rather than using radium to inline
my styles.

On Sun, Mar 13, 2016 at 9:21 AM, Dzmitry Khadorkin <notifications@github.com

**Comment by luigiplr** [comment]
This should be fixed as of 2693e9691dcf830d5185f9dbcb0a7ac7f5f79efa.

**Comment by dminkovsky** [comment]
I was thinking this might be case, but I wasn't sure why. Please report whether this is confirmed and I will close.
